ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/port-channel-interfaces.j2

```diff
@@ -32,6 +32,12 @@ interface {{ port_channel_interface }}
 {%     if port_channel_interfaces[port_channel_interface].native_vlan is arista.avd.defined and port_channel_interfaces[port_channel_interface].mode is arista.avd.defined("trunk") %}
    switchport trunk native vlan {{ port_channel_interfaces[port_channel_interface].native_vlan }}
 {%     endif %}
+{%     if port_channel_interfaces[port_channel_interface].phone.vlan is defined and port_channel_interfaces[port_channel_interface].phone.vlan %}
```

Comment:
Please use `arista.avd.defined` filter. Compared to the builtin is defined test, this test will also test for none and can even test for a specific value.